src/protocol.h

```diff
@@ -76,6 +76,18 @@ extern const char* VERACK;
  * network.
  */
 extern const char* ADDR;
+/**
+ * The addrv2 message relays connection information for peers on the network just
+ * like the addr message, but is extended to allow gossiping of longer node
```

Comment:
99047b07 nit, maybe "gossiping of longer node addresses up to 512 bytes"

Reply:
I deliberately leave numbers outside of comments - as too much unnecessary details and also could go out of sync if the code is updated and the comment not.